src/data/list/modeq.lean

```diff
@@ -0,0 +1,68 @@
+/-
+Copyright (c) 2018 Chris Hughes. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Chris Hughes
+-/
+import data.nat.modeq
+import data.list.rotate
+
+/-! # List rotation and modular arithmetic -/
```

Comment:
Can we just merge this file into `data.list.rotate`? Or are there reasons this is a bad idea?

Reply:
I did think about this, but I know Mario wants to move many of the more basic `list` files into `Std` eventually, so I don't want to increase any of their imports.